src/braket/analog_hamiltonian_simulator/rydberg/validators/hamiltonian.py

```diff
@@ -49,9 +47,7 @@ def all_sequences_in_driving_and_shifting_fields_have_the_same_last_timepoint(cl
                     -1
                 ]
         for index, field in enumerate(
-            values["shiftingFields"]
-            if "shiftingFields" in values.keys()
-            else values["localDetuning"]
+            values["localDetuning"] if "localDetuning" in values.keys() else values["localDetuning"]
```

Comment:
Same as above. This seems like it could just be `values["localDetuning"]`

Reply:
Fixed